contribs/gmf/examples/locationchooser.js

```diff
@@ -41,6 +41,13 @@ app.MainController = function() {
     extent: [12044030, -4921322, 17914393, -1007746]
   }];
 
+
+  /**
+   * @type {gmfx.LocationchooserLocation}
```

Comment:
It looks like you have to use `{gmfx.LocationchooserLocation|undefined}` here. Or you could provide an initial location, as an illustration of using `gmf-locationchooser-selected` as an input mechanism.